src/main/java/nl/tudelft/ewi/devhub/server/database/entities/issues/Issue.java

```diff
@@ -0,0 +1,42 @@
+package nl.tudelft.ewi.devhub.server.database.entities.issues;
+
+import java.net.URI;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import nl.tudelft.ewi.devhub.server.database.entities.User;
+
+
+@Data
+@Entity
```

Comment:
Please read a little bit on [JPA Inheritance](https://en.wikibooks.org/wiki/Java_Persistence/Inheritance) if you're not yet familiar with that. I am thinking whether it would be a good idea to use `InheritanceType.SINGLE_TABLE` or `InheritanceType.SINGLE_TABLE`, instead of making `AbstractIssue` a `@MappedSuperclass`. The benefit of this is that we can make foreign keys relation ships to the `AbstractIssue`, and by that means make the `PullRequestComment` an `AbstractIssueComment`, without having to introduce a separate `IssueComment` (plus a data access object for it, and a new table, etc...)